README.adoc

```diff
@@ -1,3 +1,191 @@
-# OkHttp Plugin for Jenkins
+# OkHttp API Plugin for Jenkins
+
+This plugin manages okhttp library version for Jenkins plugins.
+
+It includes the following packages:
```

Comment:
Maybe worth noting explicitly that this plugin does not and will not include `okhttp-urlconnection` for OkHttp 3.x going forward.